### PR TITLE
As a user, I want to easily access Verax documentation and website

### LINKED ISSUE #781: [US] As a user, I want to easily access Verax documentation and website
As a user,
I want to easily access Verax documentation and website,
So that I can save some time

----

Image
The 2 blocks at the bottom of the Explorer's homepage should be updated.

- [ ] The left one should link to the [Developer Guide](https://docs.ver.ax/verax-documentation/developer-guides/for-attestation-issuers) on the Documentation
- [ ] The right one should link to the [main landing page](https://www.ver.ax/)

### PR DESCRIPTION
**Pull request by Chirag-S-Kotian**
## Description
Updates the Explorer's homepage blocks to provide direct access to Verax documentation and website, improving user experience and accessibility to key resources.

## Changes
- Left block now links directly to the Developer Guide for attestation issuers
- Right block now links to the Verax main landing page

## Links
- Resolves #781

## Testing
1. Visit the Explorer's homepage
2. Verify left block's "Get Started" button links to: https://docs.ver.ax/verax-documentation/developer-guides/for-attestation-issuers
3. Verify right block's "Learn More" button links to: https://ver.ax

### CODE REVIEW

**Comment by Chirag-S-Kotian** [comment]
is it good or not? @alainncls

File: explorer/src/pages/Home/components/Info/data.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ export const infoData = [
     title: t("home.info.aboutVerax"),
     button: {
       name: t("common.actions.learnMore"),
-      url: "https://docs.ver.ax/verax-documentation/",
+      url: "https://ver.ax",
```
Comment (alainncls) [inline explorer/src/pages/Home/components/Info/data.tsx:21]:
```suggestion
      url: "https://www.ver.ax",
```

**Review by alainncls — CHANGES_REQUESTED** [review]

**Comment by alainncls** [comment]
> is it good or not? @alainncls

Just one change on the URL for the Verax website

File: explorer/src/pages/Home/components/Info/data.tsx (hunk 1 above)
Comment (Chirag-S-Kotian) [reply explorer/src/pages/Home/components/Info/data.tsx]: done

**Review by alainncls — APPROVED** [review]

**Pull request merged**